docker-compose.yml

```diff
@@ -110,6 +110,10 @@ services:
       test: redis-cli ping | grep PONG
     volumes:
       - "sentry-redis:/data"
+      - type: bind
+        read_only: true
+        source: ./redis/redis.conf
+        target: /usr/local/etc/redis/redis.conf
```

Comment:
Would you mind either mounting the entire `redis` directory or moving the `redis.conf` file to the root level?

See this comment/discussion for context: https://github.com/getsentry/self-hosted/pull/3395#discussion_r1830788235

Reply:
hi BYK, as you said I changed it to these values and paths:
```
    volumes:
      - "sentry-redis:/data"
      - type: bind
        read_only: true
        source: ./redis.conf
        target: /usr/local/etc/redis/redis.conf
```